pkg/apis/core/helper/helpers.go

```diff
@@ -392,3 +394,25 @@ func CalculateSeedUsage(shootList []*core.Shoot) map[string]int {
 
 	return m
 }
+
+// GetSecretBindingTypes returns the SecretBinding provider types.
+func GetSecretBindingTypes(secretBinding *core.SecretBinding) []string {
+	return strings.Split(secretBinding.Provider.Type, ",")
+}
+
+// SecretBindingHasType checks if the given SecretBinding has the given provider type.
+func SecretBindingHasType(secretBinding *core.SecretBinding, toFind string) bool {
```

Comment:
Should we rather use a name that suggest what it is that we are looking for and matches the doc string? Please correct other usages as well.
```suggestion
func SecretBindingHasType(secretBinding *core.SecretBinding, providerType string) bool {
```